deidentify/util/replace_phi.py

```diff
@@ -52,3 +56,16 @@ def mask_annotations(document: Document,
 
     text_rewritten += document.text[original_text_pointer:]
     return Document(name=document.name, text=text_rewritten, annotations=annotations_rewritten)
+
+
+def surrogate_annotations(docs: List[Document], seed=42) -> List[Document]:
+    random_data = RandomData(seed=seed)
+    dataset_deidentifier = DatasetDeidentifier(random_data=random_data)
+
+    surrogate_docs = [SurrogateDocument(doc.annotations, doc.text) for doc in docs]
```

Comment:
Stupid question; shouldn't this be a generator given that we're passing an entire corpus?

Reply:
You are right, that would be preferable. However, I'm afraid the current implementation of `DatasetDeidentifier` does not support generating surrogates in a streaming fashion.